autocomplete-us-separate-fields.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Form Autofill U.S. @autocomplete Demo</title>
+  <style>
+    label {
+      display: block;
+      margin-block-start: 1em;
+      margin-block-end: 1em;
+      width: 18em;
+      text-align: right;
+    }
+</style>
+</head>
+<body>
+  <h1>Form Autofill U.S. @autocomplete Demo</h1>
+
+  <form id="form">
+    <label>Given Name: <input autocomplete="given-name" autofocus></label>
+    <label>Additional Name: <input autocomplete="additional-name"></label>
+    <label>Family Name: <input autocomplete="family-name"></label>
+    <label>Organization: <input autocomplete="organization"></label>
+    <label>Address Line 1: <input autocomplete="address-line1"></label>
+    <label>Address Line 2: <input autocomplete="address-line2"></label>
+    <label>Address Line 3: <input autocomplete="address-line3"></label>
+    <label>City: <input autocomplete="address-level2"></label>
+    <label>State: <input autocomplete="address-level1"></label>
+    <label>ZIP Code: <input autocomplete="postal-code"></label>
+    <label>Country: <input autocomplete="country"></label>
+    <label>Telephone: <input autocomplete="tel-country-code" size="2"><input autocomplete="tel-area-code" size="3" maxlength="3"><input autocomplete="tel-local-prefix" size="3" maxlength="3"><input autocomplete="tel-local-suffix" size="4" maxlength="4">
+    </label>
+    <label>Email: <input autocomplete="email"></label>
+    <p>
+      <input type="submit" value="Submit">
+      <button type="reset">Reset</button>
+    </p>
+  </form>
+
+  <p><button id="btn">Add new input</button></p>
```

Comment:
I guess this section is not necessary because it was created for testing `DOMInputAdded`.